dotcom-rendering/src/lib/sendTargetingParams.ts

```diff
@@ -0,0 +1,17 @@
+import { isArray } from 'lodash';
```

Comment:
I think we could rename this module to `sendTargetingParams.apps.ts` because we won't be using this on web. The implementation below also assumes this: `set('p', 'app');`